package.json

```diff
@@ -66,16 +66,16 @@
     "ts-prune": "^0.10.3",
     "tsx": "^3.9.0",
     "turbo": "^1.6.3",
-    "typescript": "^4.8.3"
+    "typescript": "^4.8.3",
+    "vitest": "^0.28.5"
```

Comment:
This shouldnt be necessary in root?

Reply:
i guess maybe we do want the globals to be available throughout all packages 🤔

Reply:
Exactly, otherwise we can't import `vitest` in other packages

Reply:
not a fan of that - i like explicitely defining vitest as a dep whereever you need it

Reply:
otherwise we might end up putting tests in examples without adding the dep which would break on subtree